backend/public-graph/graph/resolver.go

```diff
@@ -767,7 +768,7 @@ func (r *Resolver) HandleErrorAndGroup(ctx context.Context, errorObj *model.Erro
 		settings, _ = r.Store.GetAllWorkspaceSettings(ctx, workspace.ID)
 	}
 	var embedding *model.ErrorObjectEmbeddings
-	if settings != nil && settings.ErrorEmbeddingsGroup {
+	if util.IsDevEnv() || (settings != nil && settings.ErrorEmbeddingsGroup) {
```

Comment:
Just want to double check that this is intended and not just left in from testing in dev?

Curious if we should rely on all dev envs being tied to embeddings vs allowing an engineer to edit/update a setting?